Components/ShoppingItem.js

```diff
@@ -0,0 +1,70 @@
+import styled from "styled-components";
+
+export default function ShoppingItem({ quantity, name, category }) {
+  return (
+    <ItemContainer category={category}>
+      <ItemQuantity>{quantity}</ItemQuantity>
+      <ItemName>{name}</ItemName>
+      <ItemCategory>{category}</ItemCategory>
+    </ItemContainer>
+  );
+}
+
+const ItemContainer = styled.ul`
```

Comment:
You have aleady an `ul` in your `ShoppingItemList` component, meaning your whole shopping list is the `ul`. One shopping item should be an `li` element, as the direct children from an `ul` should always be `li` elements.
```suggestion
const ItemContainer = styled.li`
```

Reply:
Thank you! Changed it.